arkestrator/wsgi.py

```diff
@@ -1,3 +1,9 @@
-import django.core.handlers.wsgi
+import django.core.wsgi
+import sys
 
-application = django.core.handlers.wsgi.WSGIHandler()
+sys.path.append("/opt/arkestrator/virtualenv/src/arkestrator")
```

Comment:
Can we at least use `__file__` here?

Reply:
I agree, but want to point out we also  have the same path in `settings.py` so it should be changed in both places.

Reply:
__file__ is just the name of the running file